code/game/objects/items/rogueweapons/melee/polearms.dm

```diff
@@ -101,12 +101,13 @@
 	clickcd = CLICK_CD_CHARGED
 
 /datum/intent/sword/bash
-	name = "pommel bash"
+	name = "crossguard bash"
```

Comment:
this should honestly remain as pommel bash, it's not very fitting to hit someone with your crossguard if you're holding your sword normally (most swords get one-handed bash), especially because the crossguard is what it is versus pommel strikes being the equivalent of a pistol whip

Reply:
We already have "pomel strike", so what you suggest is redundant.

Reply:
😛